vault/resource_kmip_secret_scope.go

```diff
@@ -115,26 +128,23 @@ func kmipSecretScopeDelete(d *schema.ResourceData, meta interface{}) error {
 func isScopeConfigured(client *api.Client, path, name string) (bool, error) {
 	resp, err := client.Logical().List(path)
 	if err != nil {
-		return false, fmt.Errorf("error reading KMIP scopes at %s: %s", path, err)
-	}
-	if resp == nil {
-		return false, fmt.Errorf("expected scopes at %s, no scopes found", path)
+		return false, fmt.Errorf("error reading KMIP scopes at %s: err=%w", path, err)
 	}
 
-	var scopes []interface{}
-	if v, ok := resp.Data["keys"].([]interface{}); ok && v != nil {
-		scopes = v
-	}
-	found := false
-	for _, s := range scopes {
-		if s.(string) == name {
-			found = true
-			break
+	if resp != nil {
+		var scopes []interface{}
```

Comment:
We can probably drop this per the comment below.

Reply:
Fixed in ab39eb6